integration/secret/secret_test.go

```diff
@@ -152,27 +151,8 @@ func TestSecretsCreate(t *testing.T) {
 		Data: []byte("TESTINGDATA"),
 	})
 	testutil.ErrorContains(t, err, "already exists")
-}
-
-func TestSecretsDelete(t *testing.T) {
-	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
-
-	defer setupTest(t)()
-	d := swarm.NewSwarm(t, testEnv)
-	defer d.Stop(t)
-	client, err := client.NewClientWithOpts(client.WithHost((d.Sock())))
-	require.NoError(t, err)
-
-	ctx := context.Background()
-
-	testName := "test_secret"
-	secretID := createSecret(ctx, t, client, testName, []byte("TESTINGDATA"), nil)
-	require.NoError(t, err)
-
-	insp, _, err := client.SecretInspectWithRaw(ctx, secretID)
-	require.NoError(t, err)
-	assert.Equal(t, insp.ID, secretID)
 
+	// Ported from original TestSecretsDelete
```

Comment:
If we're doing multiple tests in a single `Test` function then I think we should probably use `t.Run()` for each so that if one fails we still run the test. It's also self-documenting with the test name instead of an inline comment, where the scope of the comment is ambiguous.